db/migrate/20230223153716_add_receiving_only_statutory_sick_or_maternity_pay_to_employments.rb

```diff
@@ -0,0 +1,5 @@
+class AddReceivingOnlyStatutorySickOrMaternityPayToEmployments < ActiveRecord::Migration[7.0]
+  def change
+    add_column :employments, :receiving_only_statutory_sick_or_maternity_pay, :boolean, default: false
```

Comment:
null: false here is useful. Just been bitten by one of these in another story, where client explicitly (by mistake) set value to nil